### ISSUE TITLE
"the file contains a virus" on Windows 10

### ISSUE THREAD

**Issue by lukepuplett** [stripe/stripe-cli#379]
### Issue
Hello - I downloaded and unzipped `stripe_1.3.4_windows_x86_64.zip` and ran the `stripe.exe` binary inside but received:

> Operation did not complete successfully because the file contains a virus or potentially unwanted software.

![image](https://user-images.githubusercontent.com/5802524/74670648-861f3c80-51a1-11ea-95c5-fad8430625c7.png)

### Expected Behavior
I think I expected an installer to run.

### Steps to reproduce
On Windows 10 v2004 (19041.84), download and unzipp `stripe_1.3.4_windows_x86_64.zip` and run the `stripe.exe` binary, accept any warnings about running untrusted software.

### Traceback
None.

### Environment
Windows 10 v2004 (19041.84)

**Comment by lukepuplett** [comment]
Just as a side note, I also tried running the Stripe CLI from a WSL2 console on my machine but although it works, it won't talk to the web server running on the Windows side (testing webhooks).

So if anyone knows how to solve that, its an alternative for us --poor sods-- developing on Windows.

**Comment by Julienh** [comment]
Hello
Same problem, here is the Trojan detected : Trojan:Win32/Emali.A!cl

**Comment by lukepuplett** [comment]
Just to come back on using Stripe CLI under WSL2 talking to a .NET Core website on the Windows side, you'll need to:

 - Ensure Kestrel is listening on the physical IP
 - Setup a hosts entry for an alias for your physical IP
 - Setup a self-signed certificate for that hostname
 - Add a Windows Firewall rule
 - Then use `stripe --skip-verify ...`

**Comment by ob-stripe** [comment]
Hi, thanks for the report! It does look like recent versions of the CLI are reported as containing the "Trojan:Win32/Emali.A!cl" virus by Microsoft Defender. This is a false positive, and is safe to ignore. We will reach out to Microsoft to clear up the issue.

**Comment by ob-stripe** [comment]
Closing this issue, but please feel free to reply if you're still getting false positives with the most recent versions of the CLI and we'll reopen.

**Comment by R4wizard** [comment]
I've received this false positive today from Windows Defender.

**Comment by VladMstv** [comment]
Same here. After I've executed "scoop install stripe", windows defender says that it's detected Trojan:Win32/Occamy.C41

**Comment by skoggimord** [comment]
I tried downloading it today, it is blocked by Windows defender and reported as having a severe level payload of Trojan:Win32/Occamy.C41

**Comment by andre7000** [comment]
Same here...  

scoop install stripe
Installing 'stripe' (1.4.5) [64bit]
stripe_1.4.5_windows_x86_64.zip (6.2 MB) [====================================================================] 100%
Operation did not complete successfully because the file contains a virus or potentially unwanted software.

URL https://github.com/stripe/stripe-cli/releases/download/v1.4.5/stripe_1.4.5_windows_x86_64.zip is not valid

**Comment by juniormayhe** [comment]
> Closing this issue, but please feel free to reply if you're still getting false positives with the most recent versions of the CLI and we'll reopen.

@ob-stripe Did you manage to reach Microsoft about this?

```
C:\>scoop bucket add stripe https://github.com/stripe/scoop-stripe-cli.git
Checking repo... ok
The stripe bucket was added successfully.

C:\>scoop install stripe
Installing 'stripe' (1.4.5) [64bit]
stripe_1.4.5_windows_x86_64.zip (6.2 MB) [=========================================================================================================================================================================================] 100%
Move-Item : Operation did not complete successfully because the file contains a virus or potentially unwanted software.
At C:\Users\junior\scoop\apps\scoop\current\lib\install.ps1:112 char:9
+         Move-Item "$cached.download" $cached -force
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\Users\junior...64.zip.download:FileInfo) [Move-Item], IOException
    + FullyQualifiedErrorId : MoveFileInfoItemIOError,Microsoft.PowerShell.Commands.MoveItemCommand

Copy-Item : Cannot find path 'C:\Users\junior\scoop\cache\stripe#1.4.5#https_github.com_stripe_stripe-cli_releases_download_v1.4.5_stripe_1.4.5_windows_x86_64.zip' because it does not exist.
At C:\Users\junior\scoop\apps\scoop\current\lib\install.ps1:116 char:9
+         Copy-Item $cached $to
+         ~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\junior...dows_x86_64.zip:String) [Copy-Item], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyItemCommand

Checking hash of stripe_1.4.5_windows_x86_64.zip ... Resolve-Path : Cannot find path 'C:\Users\junior\scoop\apps\stripe\1.4.5\stripe_1.4.5_windows_x86_64.zip' because it does not exist.
At C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules\Microsoft.PowerShell.Utility\Microsoft.PowerShell.Utility.psm1:110 char:36
+                 $pathsToProcess += Resolve-Path $Path | Foreach-Objec ...
+                                    ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\junior...dows_x86_64.zip:String) [Resolve-Path], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.ResolvePathCommand

ERROR You cannot call a method on a null-valued expression.
ERROR Hash check failed!
App:         stripe/stripe
URL:         https://github.com/stripe/stripe-cli/releases/download/v1.4.5/stripe_1.4.5_windows_x86_64.zip
Expected:    8c7aae178d6dacc0e29ea64102b5c9ec46d7636928c5e6636539d06633b99c76
Actual:

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/stripe/scoop-stripe-cli/issues/new?title=stripe%401.4.5%3a+hash+check+failed

C:\>
```

Version: Microsoft Windows [Version 10.0.18363.959]

**Comment by auchenberg** [comment]
Hi 👋 

We have been in contact with Microsoft, and it's our understanding that our Windows builds of the CLI have been removed from the warning list.

Please let us know if that's not true.

Thanks all for reporting this!

/k

**Comment by anderswi** [comment]
@auchenberg I got it flagged as "Trojan:Win32/Emali.A!cl" by Windows defender today. Version 1.5.1 of the windows binary.

**Comment by dryhurst** [comment]
![image](https://user-images.githubusercontent.com/1443017/90076421-3f7c4c00-dcc5-11ea-9019-6a3a5414964f.png)

**Comment by auchenberg** [comment]
@dryhurst Thanks for the heads-up, and sorry about this. We are on it.

**Comment by nimeshvaghasiya** [comment]
@auchenberg Any Update on this? I'm having same issue as @dryhurst

**Comment by Sun3** [comment]
Any updates... It's June 14th 2021 and Windows 10 is still reporting the same virus....

**Comment by tomer-stripe** [comment]
We've been working on this for a while now but I think it's a bit of a whack-a-mole game with virus scanners on Windows: https://golang.org/doc/faq#virus.

I just checked 1.6.3 and it is reporting an issue with MaxScore, which is a false positive. Sadly we're not the only ones with this problem: https://github.com/lian/msfs2020-go/issues/14

**Comment by IARKI** [comment]
Same problem here

**Comment by Skoggi** [comment]
Tried downloading the cli today (3rd November 2021), blocked by windows as containing a virus.

**Comment by ben-springsit** [comment]
Having the same problem on Nov. 4th, 2021.

After installing Go on my PC, I was able to run:
`go get -u github.com/stripe/stripe-cli/...`
... in my terminal. This resolved the issue for me, and automagically added the Stripe CLI to my path. Fancy!
